packages/workbox-core/test/bundle/node/controllers/test-WorkboxCore.js

```diff
@@ -0,0 +1,34 @@
+import chai from 'chai';
```

Comment:
You should be able to do

```js
import {expect} from 'chai';
```

and then not have to assign `chai.expect` to a variable later on. The `commonjs` Rollup plugin synthesizes named exports for you: https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports